src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/SnackBar.java

```diff
@@ -31,54 +31,50 @@
+  public void navigateToHomePage() {
+    try {
+      SeleniumDriver.openPage(Constants.WRANGLE_HOME_URL);
+      WaitHelper.waitForPageToLoad();
+    } catch (Exception e) {
+      System.err.println("error:" + e);
     }
+  }
 
-    @Then("Click on the data exploration card")
-    public void clickOnTheDataExplorationCard() {
-        try {
-            WaitHelper.waitForPageToLoad();
-
-            List<String> productName = new ArrayList<String>();
-            List<WebElement> allProductsName = SeleniumDriver.getDriver().findElements(
-                    By.xpath(".//*[@data-testid='wrangler-home-ongoing-data-exploration-card']"));
-            ElementHelper.clickOnElement(allProductsName.get(0));
-            String url = SeleniumDriver.getDriver().getCurrentUrl();
-            Assert.assertTrue(url.contains("http://localhost:11011/cdap/ns/default/wrangler-grid"));
-        } catch (Exception e) {
-            System.err.println("error:" + e);
-        }
+  @Then("Click on the data exploration card")
+  public void clickOnTheDataExplorationCard() {
+    try {
+      WaitHelper.waitForPageToLoad();
+      List<String> productName = new ArrayList<String>();
+      List<WebElement> allProductsName = SeleniumDriver.getDriver().findElements(
+              By.xpath(".//*[@data-testid='wrangler-home-ongoing-data-exploration-card']"));
+      ElementHelper.clickOnElement(allProductsName.get(0));
+      String url = SeleniumDriver.getDriver().getCurrentUrl();
+      Assert.assertTrue(url.contains("http://localhost:11011/cdap/ns/default/wrangler-grid"));
+    } catch (Exception e) {
+      System.err.println("error:" + e);
     }
+  }
 
-    @Then("Click on the Snackbar close icon")
-    public void verifyTheSnackbarPopUpIsComingOrNot() {
-        try {
-            WaitHelper.waitForPageToLoad();
-            boolean flag = true;
-            while (flag == true) {
-                if (Helper.isElementExists(Helper.getCssSelectorByDataTestId("loading-indicator"))) {
-                    flag = true;
-                } else {
-                    flag = false;
-                }
-            }
-            WebElement ele = SeleniumDriver.getDriver().findElement
-                    (By.xpath("//*[@data-testid = 'snackbar-close-icon']"));
-            JavascriptExecutor executor = (JavascriptExecutor) SeleniumDriver.getDriver();
-            executor.executeScript("arguments[0].click();", ele);
-        } catch (Exception e) {
-            System.err.println("error:" + e);
+  @Then("Click on the Snackbar close icon")
+  public void verifyTheSnackbarPopUpIsComingOrNot() {
+    try {
+      WaitHelper.waitForPageToLoad();
+      boolean flag = true;
+      while (flag == true) {
+        if (Helper.isElementExists(Helper.getCssSelectorByDataTestId("loading-indicator"))) {
+          flag = true;
+        } else {
+          flag = false;
         }
+      }
+      WebElement ele = SeleniumDriver.getDriver().findElement
```

Comment:
I see, but I also want to avoid using `SeleniumDriver.getDriver().findElement` in our tests when we have helper methods for locating elements such as `Helper.locateElementByTestId(<data-testid>)`.